src/figvalidationservice/src/main/java/ca/bc/gov/open/pssg/rsbc/dps/figvalidationservice/facility/ValidateFacilityPartyResponse.java

```diff
@@ -0,0 +1,83 @@
+package ca.bc.gov.open.pssg.rsbc.dps.figvalidationservice.facility;
+
+import ca.bc.gov.open.pssg.rsbc.dps.figvalidationservice.FigaroValidationServiceConstants;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.dataformat.xml.annotation.JacksonXmlRootElement;
+
+/**
+ *
+ * Represents the Validate Facility Party Response
+ *
+ * @author alexjoybc@github
+ *
+ */
+@JacksonXmlRootElement(localName = "validateFacilityPartyResponse")
+@JsonInclude(JsonInclude.Include.NON_NULL)
```

Comment:
Does the response return empty attributes if the ORDS values are empty (empty strings)? If so, this will have to be NON_EMPTY.

Reply:
ok good catch will do it